plugins/services/src/js/structs/Service.d.ts

```diff
@@ -1,11 +1,18 @@
 import Item from "#SRC/js/structs/Item";
 import { Status } from "#PLUGINS/services/src/js/constants/ServiceStatus";
 
+interface QuotaRolesStats {
```

Comment:
Should we move this to another file instead of duplicating here and in `ServiceTree.d.ts` & `ServiceGroup`?